workers/send_sms.js

```diff
@@ -20,10 +20,18 @@ module.exports = function(sid, authToken, from) {
       return cb(new Error("Missing required `body` param"));
     }
 
+    var sendFrom = from,
+          toCountryCode = data.to.split(" ")[0].replace("+", ""),
```

Comment:
note: incoming phone numbers will all have the format `+<num><space><rest of number>`